src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/model/ItemGroupData.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.companieshouse.itemgroupworkflowapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ItemGroupData {
```

Comment:
I think this model class probably needs setters too. It makes it easier to write code that will need to update such objects. Arguably not part of DCAC-47, but definitely coming up.